guides/source/caching_with_rails.md

```diff
@@ -179,6 +179,24 @@ With `touch` set to true, any action which changes `updated_at` for a game
 record will also change it for the associated product, thereby expiring the
 cache.
 
+### Shared Partial Caching
+
+It is possible to share partials and associated caching between files with different mime types. For example shared partial caching allows template writers to share a partial between html and javascript files. Because of the way templates are collected in the template resolver file paths that only include the template language extension, not the mime type, can be used across mime types. So a request for an html file or js file that includes the following code:
```

Comment:
Can you change `between html and javascript files` to `between HTML and JavaScript files`? Our doc guidelines use proper names when applicable.

This sentence is a little confusing
> Because of the way templates are collected in the template resolver file paths that only include the template language extension, not the mime type, can be used across mime types. So a request for an html file or js file that includes the following code

What do you think about:
> When templates are collected in the template resolver file paths they only include the template language extension and not the mime type. Because of this templates can be used for multiple mime types. Both HTML and JavaScript requests will respond to the following code: